Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ocaml/opam:alpine
+RUN opam pin add -n tlstunnel git://github.com/avsm/tlstunnel && \
```

Comment:
thx!  this should be `hannesm/tlstunnel`? and using `https://` instead of `git://` (so that anyone has a chance of doing some transport layer security and authentication)